stack.yaml

```diff
@@ -5,12 +5,14 @@ resolver: lts-14.15
 packages:
 - .
 
+
 extra-deps:
 - git:
     https://gitlab.com/morley-framework/morley.git
     # ^ CI cannot use ssh, so we use http clone here
   commit:
-    01811fd2a6c9726cac287e829fd58d5fd77b3cba # master
+    # 72e696381c4283706953879e36a74152401f985b # master
```

Comment:
Last minute update: your morley MR has been merged (also the current commit does not work anymore because rebase, I think).